content/javascript/concepts/callbacks/callbacks.md

```diff
@@ -15,20 +15,29 @@ Callback functions are functions that are passed as arguments in other functions
 
 In JavaScript, functions can be passed as arguments because functions are objects.
 
-Suppose there are two functions, `functionA()` and `functionB()`:
+Suppose we want to create a function which makes a calculation and doubles the results of that calculation:
 
 ```js
-function functionA(num1, num2) {
+function doubleResult(num1, num2, calc) {
+  return calc(num1, num2) * 2;
+}
+
+function add(num1, num2) {
   return num1 + num2;
 }
 
-function functionB(callback) {
-  return callback(2, 4) * 2;
+function multiply(num1, num2) {
+  return num1 * num2;
 }
 
-console.log(functionB(functionA)); // Output: 12
+console.log(doubleResult(4, 2, add)); // Output: 12
+console.log(doubleResult(4, 2, multiply)); // Output: 16
 ```
 
```

Comment:
Can you add a code block of callback functions?

Reply:
Unfortunately I can't understand what you mean. Can you please clarify?

Reply:
@rzvl Find more detail on Codebytes [here](https://github.com/Codecademy/docs/blob/6b01926359f5f1c3d3751a52f23e36b659c78f89/documentation/content-standards.md#codebytes-runnable-code-blocks).

Reply:
I just add a codebyte example. Hope it's what you asked for...